tgui/packages/tgui/interfaces/TrackedPlaytime.js

```diff
@@ -15,6 +15,8 @@ const PlaytimeSection = props => {
   return (
     <Table>
       {sortedPlaytimes.map(([jobName, playtime]) => {
+        if(!playtime)
```

Comment:
Please use curly brackets here.

Reply:
Filtered for 0h playtimes.

![image](https://user-images.githubusercontent.com/24975989/114078675-23a36700-98a1-11eb-82b4-8b5d644f21ae.png)

![image](https://user-images.githubusercontent.com/24975989/114079846-92cd8b00-98a2-11eb-8df2-38269dfcea7d.png)